src/OrchardCore.Modules/OrchardCore.AuditTrail/Views/AuditTrailEventTags.SummaryAdmin.cshtml

```diff
@@ -4,10 +4,10 @@
     <i class="fas fa-history text-secondary" aria-hidden="true"></i>
     <a href="?q=category:@Model.Descriptor.Category" data-toggle="tooltip"
         title="@T["Category"]">@Model.Descriptor.LocalizedCategory(Context.RequestServices)</a></span>
-</span>
+
 
 <span class="badge ta-badge font-weight-normal">
     <i class="fas fa-bolt text-secondary" aria-hidden="true"></i>
     <a href="?q=event:@Model.Descriptor.Name" data-toggle="tooltip"
         title="@T["Event Name"]">@Model.Descriptor.LocalizedName(Context.RequestServices)</a></span>
```

Comment:
I would put the end tag on the proper indent on the next line

Reply:
Agreed, just updated it.